.github/workflows/test_notebook.yml

```diff
@@ -15,30 +15,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Excluding Python 3.11 is fine for now, but do you know what the issue is with it?

Reply:
Its because of enterprise not having  a build for it. Check the previous job set.

Reply:
Ah, OK. Enterprise does have a `python_requires=">=3.7, <3.11"` in its `setup.py`. Not sure why -- maybe there should be an issue there about 3.11 support. Otherwise, this PR looks good to me.